docs/architecture/blazor-for-web-forms-developers/project-structure.md

```diff
@@ -246,6 +246,20 @@ To run the Blazor WebAssembly app, choose one of the following approaches:
 
 Blazor WebAssembly apps can be debugged in both browser and Visual Studio.See [Debug ASP.NET Core Blazor WebAssembly](/aspnet/core/blazor/debug) for details.
 
+Both Blazor Server and Blazor WebAssembly apps support Hot Reload in Visual Studio. Hot Reload is a feature that automatically updates changes made to a Blazor app live, in the browser. You can toggle whether Hot Reload is enabled from its icon in the toolbar:
+
+![Hot Reload in Visual Studio 2022](media/project-structure/hot-reload-1.png)
```

Comment:
```suggestion
![Visual Studio 2022: Hot Reload menu item and icon.](media/project-structure/hot-reload-1.png)
```

The text within the `![THIS TEXT](../img.png)` image markdown syntax is what renders an `<img alt-text="">`, these values should serve as a descriptive sentence and have proper punctuation. In this case, we need a `.` and the description should be a bit more explanatory to help readers who rely on this accessibility feature.